ocamlformat_reason.opam

```diff
@@ -21,15 +21,21 @@ build: [
   ["dune" "build" "-p" name "-j" jobs]
 ]
 depends: [
+  "ocaml" {>= "4.05"}
   "base" {>= "v0.11.0"}
   "base-unix"
   "cmdliner"
-  "dune" {build}
+  "dune" {build & >= "1.1.1"}
   "fpath"
-  "ocaml-migrate-parsetree" {>= "1.0.6"}
-  "octavius"
-  "reason" {= "1.13.4"}
+  "ocaml-migrate-parsetree" {>= "1.0.10"}
+  "octavius" {>= "1.2.0"}
   "stdio"
-  "lwt" {< "3.1.0"} (* ocamlformat does not need lwt, but reason requires utop requires lwt, so exclude 3.1.0 since it does not build *)
+  "uutf"
+  "reason" {= "3.3.7"}
 ]
+
+conflicts: [
+  "lwt" {>="3.1.0"} (* ocamlformat does not need lwt, but reason requires utop requires lwt, so exclude 3.1.0 since it does not build *)
```

Comment:
This should not be required anymore IIRC - the utop wrapper was split into a separate `rtop` package.

Reply:
ack